src/jdom/bluetooth.ts

```diff
@@ -91,36 +95,41 @@ class BluetoothTransport extends JDTransport {
         this._service = await this._server.getPrimaryService(
             BLUETOOTH_JACDAC_SERVICE
         )
+        console.log(`${BLUETOOTH_JACDAC_RX_CHARACTERISTIC}, ${this._service}`)
```

Comment:
do ``console.log({ rx: BLUETOOTH_JACDAC_RX_CHARACTERISTIC, service: this._service })``

Reply:
removed, was debug, not required.

Reply:
The object syntax is a great trick for logging